Modules/Server/Sources/PocketCastsServer/Public/API/ApiServerHandler+Account.swift

```diff
@@ -173,6 +173,28 @@ public extension ApiServerHandler {
         }
     }
 
+    func obtainToken(request: URLRequest, provider: SocialAuthProvider) async throws -> AuthenticationResponse {
```

Comment:
A lot of code duplication here. But I believe at this stage is better to duplicate versus to try to be smart and change the previous one.